src/main/java/javaserver/ResponseBuilder.java

```diff
@@ -0,0 +1,66 @@
+package javaserver;
+
+import java.util.Arrays;
+import java.util.HashMap;
+
+public class ResponseBuilder {
+	private String[] OK_ROOT_REQUESTS = {"GET"};
+	private String[] OK_FORM_REQUESTS = {"GET", "POST", "PUT"};
+	private String[] OK_METHOD_OPTIONS = {"GET", "HEAD", "POST", "OPTIONS", "PUT"};
+
+	protected HashMap<String, String> responseParts;
+	protected HashMap<String, String> responseCodes;
+	protected HashMap<String, String[]> okRequests;
+	
+	ResponseBuilder() {
```

Comment:
There seems to be a lot of setup in the constructor that I think can be pulled out here. By setting the `okRequests` here, this ResponseBuilder is specific only to the routes that is pre-defined within itself. However, if you were to pass in the routes instead, it'd be a much more generic, reusable piece of code. Thoughts?